File: html_portions/bootstrap/Java Script/oops/oops.js
class=>blueprint,desgin pattern,plan,temp
object=>realworld entity







class pesron {

    setperson(name, age) {

        this.name = name;

        this.age = age

    }
    printperson() {
        console.log(this.name);      //this is used for finding instant variable
        console.log(this.age);

    }
}
var obj = new person()
obj.setperson("karthika", 23)
obj.printperson()


class Student{

    setStudent(name,age){
        this.name=name;
        this.age=age
    }
    printStudent(){
        console.log(this.name);
        console.log(this.age);
    }
}
var obj = new Student()
obj.setStudent("Aravind", 23)
obj.printStudent()

File: object_oriented_programs/oops.py
class person:
    def set_person(self,age,name):
        self.name=name
        self.age =age

    def set_person(self, age, name):
        self.name = name
        self.age = age

obj=person()
obj.set_person(25,"ram")
obj.print_person()

obj1=person()
obj1.set_person(26,"varun")
obj1.print_person()